executor/common_kvm_arm64_syzos.h

```diff
@@ -121,3 +132,24 @@ guest_handle_msr(uint64 reg, uint64 val)
 	    : [val] "r"(val), [pc] "r"(insn)
 	    : "x0", "x30", "memory");
 }
+
+// See "SMC Calling Convention", https://documentation-service.arm.com/static/5f8ea482f86e16515cdbe3c6
```

Comment:
The doc says "This calling convention cannot be used by code executing AArch32 state".
Doing AArch32 mode may be another thing to test.

Reply:
I added a note that we need AArch32 support, this is indeed a good idea.